src/pages/Comments/index.tsx

```diff
@@ -0,0 +1,166 @@
+import { Grid } from "@material-ui/core";
+import {
+    AccessTime as AccessTimeIcon,
+    Person as PersonIcon,
+    QuestionAnswer as QuestionAnswerIcon,
+} from "@material-ui/icons";
+import BoxBase from "src/components/Boxs/BoxBase";
+import CommentForm from "src/components/Comments/CommentForm";
+import DividerBase from "src/components/Dividers/DividerBase";
+import FooterBase from "src/components/Footers/FooterBase";
+import CircularProgressBase from "src/components/Progress/CircularProgressBase";
+import TypographyBase from "src/components/Typography/TypographyBase";
+
+import useCommentsList from "./hooks/useCommentsList";
+import useInsertComment from "./hooks/useInsertComment";
+import usePostDetail from "./hooks/usePostDetail";
+
+import CommentList from "src/pages/Comments/CommentsList";
+import LocalStorageUtils from "src/utils/LocalStorageUtils";
+
+const Comments = () => {

```

Comment:
I think this component contains too many mb/mt/pb/pt ... 
Maybe you need to work on layout skills? But let's wait till I see the video first

Reply:
Having too many mb/mt/pb/pt is usually a sign of unmaintainable code

Reply:
cc @binhtt13122000  @thienduc12311

Reply:
@kien123456k The spacing of this component is just way too complicated